core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -1274,7 +1274,7 @@ public SELF withClasspathResourceMapping(
     ) {
         final MountableFile mountableFile = MountableFile.forClasspathResource(resourcePath);
 
-        if (mode == BindMode.READ_ONLY && selinuxContext == SelinuxContext.NONE) {
+        if (mode == BindMode.READ_ONLY && selinuxContext == SelinuxContext.SHARED) {
```

Comment:
I'm fine updating `addFileSystemBind` but with `withClasspathResourceMapping`, I'm still thinking about it... This will promote copying resources instead of binding filesystem and current source code with those condition will do so. The only scenario to bind filesystem would be with `BindMode.READ_WRITE`. Ping to @kiview and @bsideup in case I'm missing something.